ui/compontents.go

```diff
@@ -33,6 +34,49 @@ func (page pageCommon) layoutBalance(gtx layout.Context, amount string) layout.D
 		layout.Rigid(func(gtx C) D {
 			return page.theme.Label(values.TextSize14, subText).Layout(gtx)
 		}),
+		layout.Rigid(func(gtx C) D {
+			if topnav {
+				return page.layoutUSDBalance(gtx)
+			}
+			return D{}
+		}),
```

Comment:
The app header component is probably going to be the only component that uses that design for currency conversion. I don't think you should add the dollar layout to `layoutBalance` that's used by other parts of the app. It results in the need to pass a "true" or "false" to the method by other parts of the app that would never use the layout. You can simply align the dollar balance vertically beside the DCR balance on the header, rather than adding it to the layoutBalance method.